admin/plugins.php

```diff
@@ -193,7 +193,7 @@ function perflab_render_plugin_card( array $plugin_data ) {
 					$context = $status['status'];
 
 					$action_links[] = sprintf(
-						'<a href="%s" id="deactivate-%s" aria-label="%s" style="color:red;text-decoration: underline;">%s</a>',
+						'<a href="%s" id="deactivate-%s" aria-label="%s">%s</a>',
 						add_query_arg(
```

Comment:
This should be wrapped in `esc_url()` while you're at it.

Reply:
Addressed in https://github.com/WordPress/performance/pull/927/commits/a36f6b157cc6f4a01e2ecb7f246cad1f759ceb74